lms/djangoapps/instructor/tests/test_legacy_raw_download_csv.py

```diff
@@ -155,6 +155,8 @@ def get_expected_grade_data(
                 u'ID', u'Username', u'Full Name', u'edX email',
                 u'External email', u'p3', u'p2', u'p1'
             ]
+            if get_score_max:
+                expected_data["data"][-1][-3:] = (0.0, 1), (1.0, 1.0), (0.0, 1)
```

Comment:
This is kind of frightening! Good to add a comment to explain what this is doing. Negative indexing and slicing oh my